bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Carousel.java

```diff
@@ -76,4 +76,24 @@ default boolean getAutopauseDisabled() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the value of label for Carousel component
+     *
+     * @return The value of label for Carousel component
+     * @since com.adobe.cq.wcm.core.components.models 12.9.0
+     */
+    default String getCarouselLabel() {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Returns the value of label for indicators wrapper
+     *
+     * @return The value of label for indicators wrapper
+     * @since com.adobe.cq.wcm.core.components.models 12.9.0
+     */
+    default String getIndicatorsLabel() {
```

Comment:
Do we need a separate API function for this? Can't we just base the indicators off the main carousel label and prepend / append a default string:

Label: "top holiday destinations"
Indicators: "choose a slide for top holiday destinations"

.. or something similar. I think it would make the authoring experience simpler.

In the example they just demo a generic string, that could also work:
> The set of controls is grouped in a tablist element with an accessible name provided by the value of aria-label that identifies the purpose of the tabs, e.g., "Choose slide to display."

Same would apply to the Tabs component I suppose.

Reply:
I'm not sure about the prepending/appending of default string - this would cause the aria-label of carousel to be duplicated and possibly read twice to a screen reader user and as far as I know we should avoid duplicating content from the accessibility point of view.

As far as the generic string goes it does sound good, but I'm wondering if there could be any case where author would want the label to be different.

> Same would apply to the Tabs component I suppose.

Sounds similar, but I think the case is a bit different. https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html here they suggest, that the label should describe the purpose of the set of tabs, so I guess a generic label for Tabs would not work as well as here, since the tabs in carousel have one clearly defined purpose.

Reply:
> As far as the generic string goes it does sound good, but I'm wondering if there could be any case where author would want the label to be different.

I would go with the generic string for now as per the example. We can add the indicators later to the API if there seems to be a need for it.

For Tabs, that's right we can keep the author defined label.